Sources/App/Controllers/UserController.swift

```diff
@@ -70,10 +74,7 @@ final class UserController {
     // MARK: JWT Token Generation 
     
     func generateJWTToken(_ userId: Int) throws -> String {
-        let time = ExpirationTimeClaim(date: Date().addingTimeInterval(60 * 5)) // 5 minutes
-        let payload: [Claim] = [time, SubjectClaim(string: "\(userId)")]
-        let jwt = try JWT(payload: JSON(payload), signer: HS256(key: "SIGNING_KEY".makeBytes()))
-        return try jwt.createToken()
+        return try droplet.createJwtToken(String(userId))
```

Comment:
Instead of having a whole separate function for this extension on the droplet, can you move it in to the registration and login methods?